pkg/apis/serving/v1alpha2/inferenceservice_types.go

```diff
@@ -247,7 +247,6 @@ type StatusConfigurationSpec struct {
 	Name string `json:"name,omitempty"`
 	// Host name of the service
 	Hostname string `json:"host,omitempty"`
-	Replicas int    `json:"replicas,omitempty"`
```

Comment:
Why is this gone?

Reply:
this is the field which is not used anywhere, so let's remove it?

Reply:
IIUC we signal current replicas using this field. It may be a bug if it's not updated.

Reply:
no I checked, we are not doing anything with it, so if we really need it we can add back with a proper implementation.